Bento/Renderable/FocusableView.swift

```diff
@@ -0,0 +1,25 @@
+@objc(BentoFocusableView)
+public protocol FocusableView: AnyObject {
+    @objc(bento_focus)
+    func focus()
+
+    @objc(bento_neighbouringFocusEligibilityDidChange)
+    optional func neighbouringFocusEligibilityDidChange()
```

Comment:
This needs to be documented. I don't fully understand it's use :/

[nit] possible should use US spelling (`neighboring`) to match Apple frameworks... 😷

Reply:
Done.